src/core/document.js

```diff
@@ -149,21 +159,17 @@ var Page = (function PageClosure() {
     },
 
     get view() {
-      var mediaBox = this.mediaBox;
-      var cropBox = this.getInheritedPageProp('CropBox');
-      if (!isArray(cropBox) || cropBox.length !== 4) {
-        return shadow(this, 'view', mediaBox);
-      }
-
       // From the spec, 6th ed., p.963:
       // "The crop, bleed, trim, and art boxes should not ordinarily
       // extend beyond the boundaries of the media box. If they do, they are
       // effectively reduced to their intersection with the media box."
-      cropBox = Util.intersect(cropBox, mediaBox);
-      if (!cropBox) {
+      var mediaBox = this.mediaBox, cropBox = this.cropBox;
+      if (mediaBox === cropBox) {
         return shadow(this, 'view', mediaBox);
       }
-      return shadow(this, 'view', cropBox);
+
```

Comment:
Nit: I'm not sure if we need this newline for readability, let's make the diff smaller instead :-)